### DISCUSSION TITLE
need a long time url for this channel

### DISCUSSION THREAD

**Discussion by ARAB-IPTV** [iptv-org/community#170] [Q&A]
https://www.youtube.com/watch?v=xnUq9jP3Lmo&ab_channel=AlmagharibiaTV%D9%82%D9%86%D8%A7%D8%A9%D8%A7%D9%84%D9%85%D8%BA%D8%A7%D8%B1%D8%A8%D9%8A%D8%A9

**Comment by Sphinxroot** [comment]
https://www.youtube.com/channel/UCwKS2U-DwjPZZvM6mSXPKgA/live

#EXTINF:-1 tvg-id="AlmagharibiaTV.uk" tvg-country="UK" tvg-language="Arabic" tvg-logo="https://almagharibia.tv/images/logo.jpg" group-title="News",Almagharibia TV (1080p)
https://cutt.ly/DPKx7o0
